specs.js

```diff
@@ -1,11 +1,33 @@
 /* global Given, Then, describe */
```

Comment:
Agreed. I just left them in here because it's what we had before. I wanted to focus on one thing at a time, which was to just get the tests don.e